### PR TITLE
feat: update the install guides to all work in the same way.

### PR DESCRIPTION
**Pull request by billy-the-fish**
Line up the install docs so the dovetail into the pg* setup.

### CODE REVIEW

File: _partials/_install-self-hosted-docker-based.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    1. Tune your database:
+
+       ```bash
+       sudo timescaledb-tune
+       ```
+
+    This script is included with the `timescaledb-tools` package when you install TimescaleDB.
+    For more information, see [configuration][config].
+
```
Comment (cevian) [inline _partials/_install-self-hosted-docker-based.mdx]: I am 99% sure timescaledb-tune is run automatically on startup in  `timescale/timescaledb-ha:pg16` so this whole section can be deleted

Comment (billy-the-fish) [reply _partials/_install-self-hosted-docker-based.mdx]: Thank you. Removed.

File: _partials/_install-self-hosted-homebrew-based.mdx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+1.  At the command prompt, add the TimescaleDB Homebrew tap:
+
+    ```bash
+    brew tap timescale/tap
+    ```
+
+1.  Install TimescaleDB and psql:
+
+    ```bash
+    brew install timescaledb libpq
```
Comment (jgpruitt) [inline _partials/_install-self-hosted-homebrew-based.mdx:20]: I think the libpq package is just the C library -- not the psql client. right? Or am I mistaken?

File: self-hosted/install/installation-docker.md [hunk 3]
```diff
@@ -1,83 +1,82 @@
...
-includes the
-[TimescaleDB Toolkit](https://github.com/timescale/timescaledb-toolkit),
-and support for PostGIS and Patroni. If you need the smallest possible image, use
-the `timescale/timescaledb:latest-pg14` image instead.
-</Highlight>
+## Install and configure TimescaleDB on PostgreSQL
 
-</Procedure>
+Best practice for self-hosting TimescaleDB is to install the latest version of PostgreSQL and
+TimescaleDB on a [supported platform](#supported-platforms) using the packages supplied by Timescale.
```
Comment (jgpruitt) [inline self-hosted/install/installation-docker.md]: Why are we saying that self-hosting is the best practice?

File: self-hosted/install/installation-docker.md [hunk 4]
```diff
@@ -1,83 +1,82 @@
...
+
+<SelfHostedDocker />
+
+
+## Add the TimescaleDB extension to your database
+
+For improved performance, you enable TimescaleDB on each database on your self-hosted PostgreSQL instance.
+This section shows you how to enable TimescaleDB for a new database in PostgreSQL using `psql` from the command line.
```
Comment (jgpruitt) [inline self-hosted/install/installation-docker.md]: The timescaledb extension is pre-created in the default `postgres` database in both the -ha and non-ha docker images. It also gets created by default in new databases in these images.

image

image

File: self-hosted/install/installation-docker.md [hunk 5]
```diff
@@ -1,83 +1,82 @@
...
 
-The TimescaleDB HA Docker image includes [Ubuntu][ubuntu] as its operating
-system. The lighter-weight TimescaleDB (non-HA) image uses [Alpine][alpine]. The
-commands in this section use the TimescaleDB HA image, but the steps are the
-same for both (except for the location of the data-dir, see below).
+The [TimescaleDB HA](https://hub.docker.com/r/timescale/timescaledb-ha) Docker image 
+includes [Ubuntu][ubuntu] as its operating system and offers the most complete TimescaleDB 
+experience. It includes the [TimescaleDB Toolkit](https://github.com/timescale/timescaledb-toolkit),
+and support for PostGIS and Patroni.  The lighter-weight TimescaleDB 
+(non-HA) `timescale/timescaledb:latest-pg14` image uses [Alpine][alpine].
```
Comment (jgpruitt) [inline self-hosted/install/installation-docker.md]:
```suggestion
(non-HA) `timescale/timescaledb:latest-pg16` image uses [Alpine][alpine]. 
```

Comment (jgpruitt) [reply self-hosted/install/installation-docker.md]: or link to the non-ha page https://hub.docker.com/r/timescale/timescaledb

File: self-hosted/install/installation-docker.md [hunk 6]
```diff
@@ -147,95 +146,6 @@ The link provided in these instructions is for the latest version of TimescaleDB
 on PostgreSQL 14. To find other Docker tags you can use, see the
```
Comment (jgpruitt) [inline self-hosted/install/installation-docker.md]:
```suggestion
on PostgreSQL 16. To find other Docker tags you can use, see the
```

File: self-hosted/install/installation-source.md [hunk 7]
```diff
@@ -7,279 +7,60 @@ keywords: [installation, self-hosted]
...
-Description | Enables scalable inserts and complex queries for time-series data
--[ RECORD 4 ]------------------------------------------------------------------
-Name        | timescaledb_toolkit
-Version     | 1.3.1
-Schema      | public
-Description | timescaledb_toolkit
-
-tsdb=>
-```
+And that is, you have TimescaleDB running on a database on a self-hosted instance of PostgreSQL.
```
Comment (jgpruitt) [inline self-hosted/install/installation-source.md]:
```suggestion
And that is it! You have TimescaleDB running on a database on a self-hosted instance of PostgreSQL.
```

File: self-hosted/install/installation-windows.md [hunk 8]
```diff
@@ -8,177 +8,93 @@ keywords: [installation, self-hosted, Windows]
...
-1.  Download the TimescaleDB installation `.zip` file from
-    [Windows releases][windows-releases].
-1.  Locate the downloaded file on your local file system, and extract the files.
-1.  In the extracted TimescaleDB directory, right-click the `setup.exe` file and
-    select `Run as Administrator` to start the installer.
 
-</Procedure>
+<AddTimescaleDBToDB />
+
+And that is, you have TimescaleDB running on a database on a self-hosted instance of PostgreSQL.
```
Comment (jgpruitt) [inline self-hosted/install/installation-windows.md]:
```suggestion
And that is it! You have TimescaleDB running on a database on a self-hosted instance of PostgreSQL.
```

File: self-hosted/install/installation-macos.md [hunk 9]
```diff
@@ -5,197 +5,82 @@ products: [self_hosted]
...
 
-1.  At the `psql` prompt, create an empty database named `tsdb`:
 
-    ```sql
-    CREATE database tsdb;
-    ```
+<AddTimescaleDBToDB />
 
-1.  Connect to the `tsdb` database that you created:
+And that is, you have TimescaleDB running on a database on a self-hosted instance of PostgreSQL.
```
Comment (jgpruitt) [inline self-hosted/install/installation-macos.md]:
```suggestion
And that is it! You have TimescaleDB running on a database on a self-hosted instance of PostgreSQL.
```

File: self-hosted/install/installation-docker.md [hunk 10]
```diff
@@ -1,83 +1,82 @@
...
-[Docker's information about the UFW flaw](https://www.techrepublic.com/article/how-to-fix-the-docker-and-ufw-security-flaw/).
-</Highlight>
 
-When you have completed the installation, you need to configure your database so
-that you can use it. The easiest way to do this is to run the `timescaledb-tune`
-script, which is included with the `timescaledb-tools` package. For more
-information, see the [configuration][config] section.
+<AddTimescaleDBToDB />
+
+And that is, you have TimescaleDB running on a database on a self-hosted instance of PostgreSQL.
```
Comment (jgpruitt) [inline self-hosted/install/installation-docker.md]:
```suggestion
And that is it! You have TimescaleDB running on a database in a Docker container.
```

File: self-hosted/install/installation-docker.md [hunk 11]
```diff
@@ -1,83 +1,82 @@
...
 If you have already installed PostgreSQL using a method other than the pre-built
-container provided here, you could encounter errors following these
-instructions. It is safest to remove any existing PostgreSQL installations
-before you begin. If you want to keep your current PostgreSQL installation, do
-not install TimescaleDB using this method.
-[Install from source](/self-hosted/latest/install/installation-source/)
-instead.
+container provided here, you may encounter errors
+following these install instructions. Best practice is to full remove any existing PostgreSQL
+installations before you begin.
```
Comment (jgpruitt) [inline self-hosted/install/installation-docker.md]: I'm not sure this advice is valid, or at least maybe it should be worded differently. Part of the benefit of the docker approach is being able to run many database instances on the same machine simultaneously. In my experience, the most frequent issue is trying to reuse the same default port (5432) for multiple instances. That fails. Maybe we should mention that here instead?

**Review by jgpruitt — CHANGES_REQUESTED** [review]

File: self-hosted/install/installation-docker.md (hunk 3 above)
Comment (billy-the-fish) [reply self-hosted/install/installation-docker.md]: Nice point, will update for all platforms.

File: self-hosted/install/installation-docker.md (hunk 11 above)
Comment (billy-the-fish) [reply self-hosted/install/installation-docker.md]: Removed, added comment about the port when the container is started.

File: self-hosted/install/installation-docker.md (hunk 4 above)
Comment (billy-the-fish) [reply self-hosted/install/installation-docker.md]: Oooh, nice. Updated.

File: _partials/_install-self-hosted-homebrew-based.mdx (hunk 2 above)
Comment (billy-the-fish) [reply _partials/_install-self-hosted-homebrew-based.mdx:20]: libpq has the psql client as well: https://www.timescale.com/blog/how-to-install-psql-on-mac-ubuntu-debian-windows.

Comment (jgpruitt) [reply _partials/_install-self-hosted-homebrew-based.mdx:20]: Oh, cool. TIL

**Review by jgpruitt — APPROVED** [review]

**Pull request merged**